src/roadmap/Roadmap.tsx

```diff
@@ -28,29 +29,30 @@ export const Roadmap = () => {
 
   return (
     <section className="roadmap">
-      <div className="roadmap__text">
-        <span>Roadmap</span>
-        <span>View</span>
-      </div>
-      <div className="roadmap__list">
-        {statuses.map((status: Status, idx: number) => {
-          const dotClass = classNames({
-            planned: status.status === STATUSES.PLANNED,
-            'in-progress': status.status === STATUSES.IN_PROGRESS,
-            live: status.status === STATUSES.LIVE,
-            roadmap__dot: true,
-          });
-          return (
-            <div className="roadmap__item" key={idx}>
-              <div className="roadmap__status">
-                <div className={dotClass} />
-                {status.status}
-              </div>
-              <div>{status.count}</div>
+      <ul className="roadmap__text">
```

Comment:
I'm not sure a `ul` is the proper tag here, to wrap an H1 and a Link. Is there something better?

Reply:
Should I put it in a span? MDN says this "The <span> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element is a generic inline container for phrasing content, which does not inherently represent anything. It can be used to group elements for styling purposes (using the [class](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#class) or [id](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#id) attributes), or because they share attribute values, such as [lang](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#lang). It should be used only when no other semantic element is appropriate."

Reply:
Well the reason I say this is because the child of a `ul` should be an `li` and here you have children of the `ul` as `h1 and Link`

Reply:
If anything I think you need to keep the `ul` wrapping the `li` elements in your map below and use another block level element (if needed)

Reply:
Did you forget to push? The child of the `<ul>` is still a `h1` tag